.github/workflows/upload-and-deploy.yaml

```diff
@@ -132,11 +132,13 @@ jobs:
       - name: Install additional python dependency
         run: |
           pipenv install ecs_logging~=2.0
+          pipenv install pytest-datadir
 
       - name: Test lambda scripts with pytest
         run: |
           pipenv run python -m pytest tests/test_s3_event_config_lambda.py -v
           pipenv run python -m pytest tests/test_s3_to_glue_lambda.py -v
+          pipenv run python -m pytest -v tests/test_lambda_raw.py
```

Comment:
Nit: Depending on if this gets merged first or mine (may be a slight merge conflict), but i think you can link the tests like:

```
- name: Test scripts with pytest (lambda, etc.)
        run: |
          pipenv run python -m pytest \
            tests/test_s3_event_config_lambda.py \
            tests/test_s3_to_glue_lambda.py \
            tests/test_lambda_dispatch.py \
            tests/test_consume_logs.py \
            tests/test_lambda_raw.py -v
```

Reply:
Eh, it's better for the commands to be discrete and explicit, imo. I'm not sure if the behavior is the same running the tests one at a time (separate `pipenv run...`) vs running all the tests at one time (a single `pipenv run...`). But if there are advantages (other than brevity) I'm open to running all the tests at one time.